test/builtins/Transformers.jl

```diff
@@ -140,9 +140,54 @@ end
     @test Xnew[4] == X[4]
     @test MLJBase.std(Xnew[5]) ≈ 1.0
 
+    # test on ignoring a feature, even if it's listed in the `features`
+    stand.ignore = true
+    f,   = MLJBase.fit(stand, 1, X)
+    Xnew = MLJBase.transform(stand, f, X)
+
+    @test issubset(Set(keys(f)), Set(Tables.schema(X).names[[5,]]))
+
+    Xt = MLJBase.transform(stand, f, X)
+
+    @test Xnew[1] == X[1]
+    @test Xnew[2] == X[2]
+    @test Xnew[3] == X[3]
+    @test Xnew[4] == X[4]
+    @test Xnew[5] == X[5]
+
     stand = Standardizer(features=[:x1, :mickey_mouse])
     @test_logs (:warn, r"Some specified") MLJBase.fit(stand, 1, X)
 
+    stand.ignore = true
+    @test_logs (:warn, r"Some specified") MLJBase.fit(stand, 1, X)
+
+    @test_throws ArgumentError Standardizer(ignore=true)
+
+    stand = Standardizer(features=[:x4], count=true)
+    f,   = MLJBase.fit(stand, 1, X)
+    Xnew = MLJBase.transform(stand, f, X)
+
+    @test issubset(Set(keys(f)), Set(Tables.schema(X).names[[4,]]))
+
+    Xt = MLJBase.transform(stand, f, X)
+
+    @test Xnew[1] == X[1]
+    @test Xnew[2] == X[2]
+    @test Xnew[3] == X[3]
+    @test elscitype(X[4]) == Count
+    @test MLJBase.std(Xnew[4]) ≈ 1.0
+    @test Xnew[5] == X[5]
+
+    stand = Standardizer(features= !=(:x5))
```

Comment:
The function `!=(:x5)` is only available on Julia >= 1.2. Please change to `x -> !=(x, :x5)` instead